testing/tests/DevExpress.ui.widgets/treeViewParts/initialization.js

```diff
@@ -1,8 +1,23 @@
 /* global initTree */
+import TreeViewTestWrapper from "../../../helpers/TreeViewTestHelper.js";
+const createInstance = (options) => new TreeViewTestWrapper(options);
 
 QUnit.module("Initialization");
 
 QUnit.test("Init tree view", function(assert) {
     var $treeView = initTree();
     assert.ok($treeView);
 });
+
+['dataSource', 'items'].forEach((optionName) => {
+    QUnit.test(`Tree initialized even with redirect keys in ${optionName} option`, function(assert) {
+        let options = { dataStructure: "plain" };
+        options[optionName] = [
+            { id: 1, text: "item1", parentId: 3 },
```

Comment:
There are more initial configs with expanded/selected values. Do we need to check them?

Reply:
It also helps to find a bug in iterateChildren.
Thanks